### PR TITLE
[finance_report_test_and_doc] feat(runtime): migrate smoke/health ACs → package roadmap; draft→active (#1554 Step 2)

### PR DESCRIPTION
**Pull request by wangzitian0**
**Step 2 (migrate)** of the runtime package refactor (#1554), per `common/meta/migration-standard.md`: EPIC ACs for this concern are absorbed into the package; a lingering mirror = NOT migrated.

## What
- **Home the smoke/health ACs** into `common/runtime/contract.py` roadmap:
  - EPIC-008 AC8.1.1–.4 → `AC-runtime.1.*` (smoke / service reachability / DB connectivity)
  - EPIC-007 AC7.7.1–.2 → `AC-runtime.7.*` (`/health` dependency-presence)
  - each `test=` resolves to its existing proof; CODE-ONLY ⇒ `proof_kind=exact`.
- **Delete the migrated EPIC rows**; replace each section with a "migrated to the runtime package" disclaimer referencing the new ids.
- **Retag** the anchoring tests' docstrings to the package ids (so registry→tests traceability resolves).
- **Promote `runtime` draft → active** (has units + roadmap); remove from `draft-package-baseline.json`.

## Verify (all local gates green)
`check_package_contract` (6 roadmap ACs resolve), `generate_ac_registry --check`, `build_ac_traceability --check`, `lint_doc_consistency`, `check_epic_package_dual`, `check_authority_reconcile` (CODE-ONLY, code=6 llm=0), `check_draft_packages`. 43 runtime/boot tests pass; ruff + mypy clean.

## Next
**Step 3 (cleanup)**: internalize `env_smoke_test.md` prose → `common/runtime/readme.md`, **delete** the doc + retire its MANIFEST entry, and add a guard test asserting zero runtime-exclusive content remains in EPIC/SSOT.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: common/runtime/contract.py [hunk 1]
```diff
@@ -6,27 +6,29 @@
...
+shipped the ``base`` value language + dependency manifest + the
+``DependencyCheck`` port; the *switch* phase added the ``extension`` probe
+adapters (``DatabaseCheck`` / ``ObjectStorageCheck`` / ``LlmCheck``, published
+below) that ``boot.Bootloader`` delegates to; the *cleanup* phase dropped the
+silent ``skipped`` status. The *migrate* phase homes the smoke-test / health
+ACs here: EPIC-008 AC8.1.1–.3 → ``AC-runtime.1.*`` (smoke / service reachability)
+and EPIC-007 AC7.7.1–.2 → ``AC-runtime.7.*`` (``/health`` dependency-presence),
+each ``test=`` resolving to its existing proof; the package tier (CODE-ONLY)
+gives ``proof_kind=exact``. The env-smoke-test SSOT prose is absorbed into
+``readme.md``. Remaining as a future feature (not this migration): manifest-driven
```
[the remark this answers is not in the document]
Comment (wangzitian0) [reply common/runtime/contract.py]: Fixed in 9465f500. Docstring now says the migrated range is AC8.1.1–.4 (incl. DB connectivity), and I removed the premature claim that the env-smoke-test prose is absorbed — that is Step 3.

File: common/runtime/contract.py [hunk 2]
```diff
@@ -6,27 +6,29 @@
...
-from common.meta.package_contract import PackageContract
+from common.meta.package_contract import ACRecord, PackageContract
 
 CONTRACT = PackageContract(
     name="runtime",
     klass="kernel",
-    status="draft",
+    status="active",
     tier="CODE-ONLY",
     depends_on=[],
```
[the remark this answers is not in the document]
Comment (wangzitian0) [reply common/runtime/contract.py:34]: Fixed in 9465f500. readme.md Status is now 'active' (matching the contract), and the 'Public vs internal' section lists the published adapters.

File: docs/project/EPIC-008.testing-strategy.md [hunk 3]
```diff
@@ -156,14 +156,15 @@ job inventories or scenario counts into this EPIC.
...
-| AC8.1.3 | Frontend service reachable | `test_frontend_api_proxy_reachable()` | `e2e/test_core_journeys.py` | P0 |
-| AC8.1.4 | Database connectivity | `test_database_connectivity()` | `e2e/test_core_journeys.py` | P0 |
+### AC8.1: Smoke Tests (Health Checks) — migrated to the `runtime` package
+
+> The smoke-test / health-check ACs (were `AC8.1.*`) moved into the
+> `runtime` package roadmap (`common/runtime/contract.py`) under the
+> package-scoped `AC-runtime.<group>.<seq>` id scheme —
+> `generate_ac_registry.py` reads package-contract roadmaps. Migrated ids
+> (homed in the package roadmap): `AC-runtime.1.1` · `AC-runtime.1.2` ·
+> `AC-runtime.1.3` · `AC-runtime.1.4`. `runtime` owns the environment smoke test
```
[the remark this answers is not in the document]
Comment (wangzitian0) [reply docs/project/EPIC-008.testing-strategy.md:166]: Fixed in 9465f500. The stray old ids are homed: test_homepage_loads now proves AC-runtime.1.3; the two test_test_lifecycle namespace tests were mis-tagged AC8.1.1 (they test test-isolation, not health) and are de-tagged. No AC8.1.*/AC7.7.* left in tests.
